app/controllers/candidate_interface/other_qualifications/base_controller.rb

```diff
@@ -1,7 +1,17 @@
 module CandidateInterface
   class OtherQualifications::BaseController < CandidateInterfaceController
     def new
-      @qualification = OtherQualificationForm.new
+      qualifications = OtherQualificationForm.build_all_from_application(current_application)
+
+      @qualification = if qualifications.any?
+                         OtherQualificationForm.new(
+                           qualification_type: qualifications.last.qualification_type,
+                           institution_name: qualifications.last.institution_name,
+                           award_year: qualifications.last.award_year,
+                         )
```

Comment:
I'm guessing you can't just do `@qualification = qualifications.last` because you want `@qualification` to have empty values for the attributes you are not specifying?

Reply:
Yup! We only want to autofill the specified attributes as per the prototype which are type, institution and year.